tests/basic.json

```diff
@@ -200,6 +210,17 @@
         1
       ]
     },
+    {
+      "name": "multiple selectors, spaces",
```

Comment:
This should already be checked in _/tests/whitespace/selectors.json_.  it's probably two separate tests, one for each space.

Reply:
You are right, sorry I had overlooked those tests.